### PR TITLE
CI: compute individual binary sizes

### PR DESCRIPTION
**Pull request by miles170**
We need this PR to be merged before checking the binary size. See #4693

### CODE REVIEW

File: .github/workflows/CICD.yml [hunk 1]
```diff
@@ -572,15 +572,33 @@ jobs:
       shell: bash
       run: |
         ## Compute uutil release sizes
-        SIZE=$(du -s target/size-release/usr/local/bin/|awk '{print $1}')
-        SIZE_MULTI=$(du -s target/size-multi-release/usr/local/bin/|awk '{print $1}')
+        DATE=$(date --rfc-email)
+        # Skip the '[' (tests) binary
```
Comment (sylvestre) [inline .github/workflows/CICD.yml:576]: why do you skip it ? 
[ is legit

**Comment by sylvestre** [comment]
A few comments
* Looks like they all have the same sha:
* could you please sort the program by order ?
* could you please store the size in byte ? (I guess it is kb here)

    "tr": {
      "sha": "ff3a35bb658665667961061acafeff72ea9230be",
      "size": 784
    },
    "tee": {
      "sha": "ff3a35bb658665667961061acafeff72ea9230be",
      "size": 744
    },
    "cp": {
      "sha": "ff3a35bb658665667961061acafeff72ea9230be",
      "size": 1048
    },

**Comment by miles170** [comment]
> A few comments
> 
> * Looks like they all have the same sha:
> * could you please sort the program by order ?
> * could you please store the size in byte ? (I guess it is kb here)
>   "tr": {
>   "sha": "ff3a35bb658665667961061acafeff72ea9230be",
>   "size": 784
>   },
>   "tee": {
>   "sha": "ff3a35bb658665667961061acafeff72ea9230be",
>   "size": 744
>   },
>   "cp": {
>   "sha": "ff3a35bb658665667961061acafeff72ea9230be",
>   "size": 1048
>   },

1. Would you want me to put sha in top order? Something like the following

```json
"sha": "ff3a35bb658665667961061acafeff72ea9230be",
"tr": 784,
"tee": 744,
"cp": 1048,
```

2. Should I also change the `size-result.json` size unit to byte?

**Comment by sylvestre** [comment]
I still see the sha for each program 

    "arch": {
      "sha": "8e7143d503f701e2d5bc6b9c3fe475ac459a9e9f",
      "size": 684
    },

**Comment by miles170** [comment]
> I still see the sha for each program
> 
> ```
> "arch": {
>   "sha": "8e7143d503f701e2d5bc6b9c3fe475ac459a9e9f",
>   "size": 684
> },
> ```

The sha is the commit hash, not the binary hash. See https://github.com/uutils/coreutils/pull/4694#issuecomment-1495905616

**Comment by miles170** [comment]
> I still see the sha for each program
> 
> ```
> "arch": {
>   "sha": "8e7143d503f701e2d5bc6b9c3fe475ac459a9e9f",
>   "size": 684
> },
> ```

I apologize for my carelessness.

**Comment by miles170** [comment]
I checked the size result, and this PR should be ready to be merged.

**Comment by sylvestre** [comment]
perfect, thanks
```
{
  "Sat, 15 Apr 2023 11:21:58 +0000": {
    "sha": "eafa220845212a6393008b419ff063b70e5d3175",
    "sizes": {
      "[": 724,
      "arch": 684,
      "base32": 808,
      "base64": 808,
      "basename": 708,
      "basenc": 820,
      "cat": 736,
      "chgrp": 776,
      "chmod": 740,
      "chown": 784,
[...]
```

**Comment by sylvestre** [comment]
it would be nice to graph the evolution of each binaries over time :)
https://github.com/uutils/coreutils-tracking

**Comment by miles170** [comment]
> it would be nice to graph the evolution of each binaries over time :) https://github.com/uutils/coreutils-tracking

I can work on this.

**Pull request merged**